src/cobra/io/mat.py

```diff
@@ -705,7 +753,9 @@ def from_mat_struct(
             new_names[new_names.index(old_field)] = new_field
             m.dtype.names = new_names
 
-    model = Model()
+    model = (
```

Comment:
This formatting looks weird.

Reply:
I'll remove the TODO, and  fix the formatting